specification/datashare/resource-manager/Microsoft.DataShare/stable/2020-09-01/examples/DataSetMappings_SynapseWorkspaceSqlPoolTable_Create.json

```diff
@@ -0,0 +1,61 @@
+{
+  "parameters": {
+    "subscriptionId": "0f3dcfc3-18f8-4099-b381-8353e19d43a7",
+    "resourceGroupName": "SampleResourceGroup",
+    "accountName": "consumerAccount",
+    "shareSubscriptionName": "ShareSubscription1",
+    "dataSetMappingName": "datasetMappingName1",
+    "api-version": "2020-09-01",
+    "dataSetMapping": {
+      "kind": "SynapseWorkspaceSqlPoolTable",
+      "properties": {
+        "dataSetId": "3dc64e49-1fc3-4186-b3dc-d388c4d3076a",
+        "synapseWorkspaceSqlPoolTableResourceId": "/subscriptions/0f3dcfc3-18f8-4099-b381-8353e19d43a7/resourceGroups/SampleResourceGroup/providers/Microsoft.Synapse/workspaces/ExampleWorkspace/sqlPools/ExampleSqlPool/schemas/dbo/tables/table1"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Date": "Wed, 13 Sep 2017 18:04:32 GMT",
+        "x-ms-request-id": "d5496da4-9c52-402f-b067-83cc9ddea888",
+        "X-Content-Type-Options": "nosniff",
+        "x-ms-ratelimit-remaining-tenant-reads": "14999",
+        "x-ms-correlation-request-id": "25c78f97-0b0a-4fe9-ad39-883a482265cd"
+      },
+      "body": {
+        "kind": "SynapseWorkspaceSqlPoolTable",
+        "properties": {
+          "dataSetId": "3dc64e49-1fc3-4186-b3dc-d388c4d3076a",
+          "synapseWorkspaceSqlPoolTableResourceId": "/subscriptions/0f3dcfc3-18f8-4099-b381-8353e19d43a7/resourceGroups/SampleResourceGroup/providers/Microsoft.Synapse/workspaces/ExampleWorkspace/sqlPools/ExampleSqlPool/schemas/dbo/tables/table1",
+          "dataSetMappingStatus": "Ok",
+          "provisioningState": "Succeeded"
+        },
+        "id": "/subscriptions/4e745bb7-c420-479b-b0d6-a0f92d48a227/resourceGroups/SampleResourceGroup/providers/Microsoft.DataShare/accounts/consumerAccount/shareSubscriptions/ShareSubscription1/dataSetMappings/datasetMappingName1",
+        "name": "datasetMappingName",
+        "type": "Microsoft.DataShare/DataSetMapping"
```

Comment:
>icrosoft.DataShare/DataSetMapping [](start = 18, length = 33)

this should be the fully qualified resource type i.e. in this case - Microsoft.DataShare/accounts/shareSubscriptions/dataSetMappings .. I realize it was done this way in the previous api-version too .. please fix it for this api-version

Reply:
@pilor The fully qualified resource type change is recent and we're also wondering if the resource for the existing api versions should be updated too?

Reply:
The resource type that corresponds with that resource ID (and the type that RBAC will be checked on) is Microsoft.DataShare/accounts/shareSubscriptions/dataSetMappings.  Anything else is incorrect

Reply:
@pilor updated.